torch/quantization/quantize.py

```diff
@@ -333,3 +333,23 @@ def swap_module(mod, mapping):
         if type(mod) in mapping:
             new_mod = mapping[type(mod)].from_float(mod)
     return new_mod
+
+def dump_tensor(mod, prefix, target_dict):
```

Comment:
I don't think we need to expose prefix to user, we can default to '' and user can just call it with dump_tensor(mod, target_dict)